### ISSUE TITLE
[CAIA intake] Lab and Test Results domain

### ISSUE THREAD

**Issue by alexiawunder** [department-of-veterans-affairs/va.gov-team#53759]
## What does your team need support for? Check all that apply.
- [ ] Launching one or more unauthenticated pages, online forms, apps, or tools on VA.gov
- [ ] Making revisions to one or more existing VA.gov pages
- [ ] Something else 
- [x] Launching a new online form, app, or tool on VA.gov
 
## Will this new product be released incrementally (for instance 25% of users initially)? 
  - [x] Yes
  - [ ] No
 
**Note:** If you checked yes, we'll reach out to discuss details about the content in the react widget.
  
## When do you expect to launch your product to 100% of users?
  TBD, but current estimate is 2024.

  
## Supporting artifacts
*Please provide supporting artifacts as available.*

- Link to product outline: [Medical Records Product Outline](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/health-care/digital-health-modernization/mhv-to-va.gov/medical-records)
- [Link to Lab and Test Results Domain Brief](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/medical-records/data-domains/labs-and-tests/labs-and-tests-brief.md)
- Link to or attach user flows: [Lab & Tests User Flow](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/99B84A0C-F7F8-4C23-86DA-64D9DFDE2F5C)
- Link to prototype or mockups: I've only included mobile screens for brevity, but the desktop screens are available in the [Lab and Test Results section of the main Sketch File](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/p/A47501F8-078B-42CF-977B-7BF9A31B3976/canvas#Version). 
     - [Lab and test results list view](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/5E6C5D42-F939-4001-A7F2-50123AF7CFA8)
     - Chemistry/Hematology details page ([collapsed accordion](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/225256C0-861E-4F97-BA32-312CAE53864D)) and ([expanded accordion](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/E41A6860-3A9A-4648-B88B-4F39FEC703E2))
     - [Microbiology details page](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/9488FCA1-44F8-4962-AFC9-87445C40BA3E)
     - [Pathology reports details page](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/E337C4A2-59F6-447F-A299-651A9B00A79C)
     - [Radiology Results details page](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/B91B182F-4F41-47FB-A750-8CB720114CC4) (Note: this one will include an additional image element - we will update it as soon as we figure out a couple of blockers)
     - [Historical EKG Report details page](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/CA547596-64DE-4124-AAEE-49EB10C32BAC)

- Link to any epics/issues that may be helpful: TBD

## Will this work be going through the Collaboration Cycle?
- [x] Yes 
- [ ] No

## When does this work need to be done?
- Estimated launch date: TBD - this is only one domain out of the larger medical records product.
- Estimated staging review date: TBD - this is only one domain out of the larger medical records product.
- Content and IA work needed by: We'd like to conduct usability testing with approved content, which should happen late February at the earliest.

## Do you plan to bring this to an upcoming content office hours session?
- [ ] Yes, benefit content office hours (Thursdays, 3:00 p.m. to 3:45 p.m. ET)
- [ ] Yes, health  content office hours (Thursdays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] Yes, unauth office hours (Mondays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] No, but I'd like to schedule time to talk about this request
- [x] No, let's work asynchronously and meet if needed

**Note:** If we think this work would benefit from a collaborative session with you, we may ask you to bring it to office hours or set up a separate time to meet.

## About your team

- Team name: mhv-medical-records
- OCTO-DE product owner: Lauren Alexanderson
- Product manager: Bryan Riley
- Designer: Alexia Wunder, Bobby Bailey
- FE engineer: Mike Moyer
- Product/team Slack channel: #mhv-medical-records

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag Randi Hecht.**

If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)

```[tasklist]
### Tasks
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/57857
```

**Comment by laurwill** [comment]
Hi @alexiawunder , as we chatted, collecting some higher-level questions here for labs and tests. 

- Q: Can we use lab test (like "complete blood count" or "mammogram") as the card header in list view and H1 on detail page, rather than using the lab type (like "chemistry/hematology" or "radiology")?
A: Maybe, need to confirm
Follow up: If we can't flip these fields, it will be very hard for users to find the results they're looking for in the lists. We'll need to find some other way to help them find the right test. We can try to rename the lab types to be more useful, but that will still leave lots of identical results in the list views.

- Q: Can we adjust how the test names display for ease of reading and plain language? For example, can we spell out as "CBC (complete blood count)" rather than just "CBC"? 
A: Maybe, need to confirm (designs show spelled out, but may not be possible to do this)
Follow up: If we aren't able to adjust every single test name, could we tackle the X most common test names for phase 0? 

- Q: Can we display spelled-out versions of the units for results, like "XX mg/dl (milligrams per deciliter of blood)"?

- Q: "Results" field for several types seems to repeat info from other fields, like dates and locations. Can we avoid this overlap/repetition?

- Q: Disclaimers for lab and test results include a patient safety message with crisis line info and a link to make same-day mental health appointments. (I'm assuming this is related to Veterans seeing upsetting test results without their provider there to explain them.) We need to decide how to handle this part of the disclaimer content. Has this come up as a requirement for lab results domain?

**Comment by AnneCostello** [comment]
Hello! I understand that EKGs will only include historical data and will not be updated moving forward. Is there a plan for some text on the EKG details page to let Veterans know that? 
![image](https://user-images.githubusercontent.com/116298030/228044679-664f7c37-3994-4879-9287-15896203a77b.png)

**Comment by laurwill** [comment]
@AnneCostello thanks for flagging that! It turns out that not only do we no longer pull in EKG records, the ones we have are empty. They have dates and locations, but no results. So this will need a note! We'll explain that you need to do a complete medical record request from your facility in order to get EKG results. It's on my list to add this.

**Comment by laurwill** [comment]
@coultonbunney-usds @BobbyBaileyRB , documenting here some updates from my sync with @alexiawunder about lab type/lab test/test name labels. We think the word "lab" is causing some confusion on the [detail pages in this domain](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/nRvJlWp#Comment). For example, "lab details" could sound like it's details about the location that analyzed the sample, when it also includes info about the sample itself. So in addition to the other labeling suggestions from the data fields table, we're suggesting these new changes:
- Change the H2s from "Lab details" and "Test results" to "Details about this test" and "Results"
- Change "Lab type" to "Type of test"

And a couple of future considerations that research might help with: 
- Would pulling in the "Sample tested" field to cards in list view help Veterans find the results they're looking for? So they might see "Cheek swab" in addition to a confusing test name, which might help them remember what test it was.
- We've kept the very not PL "Type of test" labels for now (like "Chemistry and hematology"). Would changing all the non-radiology types to something more general like "Type of test: Lab test" be less confusing? Or are these more specific types helpful here?

**Comment by coultonbunney-usds** [comment]
> Would changing all the non-radiology types to something more general like "Type of test: Lab test" be less confusing? Or are these more specific types helpful here?

I'd be very hesitant to change the actual content of any medical records data. For example, while Lab test instead of Chemistry/hematology might be more plain language, it is changing medical data which could lead to unforeseen issues. I think we should avoid that.

**Comment by laurwill** [comment]
@alexiawunder @BobbyBaileyRB updates for the lab & test domain:
- Intro text and "What to know" text in [Medical records working content draft](https://dvagov-my.sharepoint.com/:w:/g/personal/laura_willwerth_va_gov/Eacz1GNGqrdNn3b5gC1j0n4BjNyZ6LyafJCpEfebp-cPVg?e=f6e3Hy)
- Data field labels are in github table and comments are in mobile versions of Sketch screens
- 3 larger pieces of content noted in Sketch for detail views: "Need help understanding your results?" on [chem/hem details](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/nRvJlWp) but applies to all lab/test detail screens except for EKG, "What's the standard range?" on [chem/hem details](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/Qb2Oarw?annotation=20ccfe03-95aa-45a9-97b3-de4e937612f2), and a note about how to get EKG results on [EKG details](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/4aR2Ovq?annotation=2d98061f-6434-41fc-933e-311666cdd61c)

**Note about safety message:**
- BB disclaimer for labs & tests has crisis line 988 number (for upsetting results). We hadn't included this in the "What to know" section because the universal VA.gov header with crisis line will be visible **and** we were planning to link to SM for questions, which would take users through the safety message as part of the flow. Now that we're linking to legacy SM for questions, phase 0 users won't see the SM safety message flow, but they will still see the crisis line in the universal header. If that's not sufficient, we can add a brief safety message to "What to know" or the "Need help understanding your results" content.

**Remaining questions for phase 0:**
- **Interpretation field:** What exactly goes in this field? Who enters it, the provider or the lab tech or someone else? How is it different from the "comments" field?
- **Flags:** What are all the possible flag terms in results? How are users meant to understand this info? Are there next steps associated with a flagged result?
- **EKGs:** Do we have "Ordered by" data for EKGs? List card includes provider, but EKG details page does not.

**Future work:**
- Adjusting test names and units for plain language (or providing links to resources to help Veterans understand this medical info)
- Potentially pulling in "sample tested" field to list cards to help differentiate results 

cc @aliyahblackmore @AnneCostello @coultonbunney-usds

**Comment by sara-amanda** [comment]
## CAIA Update

- **[A11y Image Review]: MHV Lab and Test Results #57857** 
    - Moved to backlog
    - Not in P0, possible in P1, revisit likely in October 2023.

**Comment by kristinoletmuskat** [comment]
Hey @alexiawunder any reason we still need this ticket open?

**Comment by alexiawunder** [comment]
@kristinoletmuskat Nope I think this work is long done.
